apps/labrinth/src/auth/email/mod.rs

```diff
@@ -34,9 +35,20 @@ pub fn send_email_raw(
     let username = dotenvy::var("SMTP_USERNAME")?;
     let password = dotenvy::var("SMTP_PASSWORD")?;
     let host = dotenvy::var("SMTP_HOST")?;
+    let port = dotenvy::var("SMTP_PORT")?.parse::<u16>().unwrap_or(25);
     let creds = Credentials::new(username, password);
+    let tls_settings =
+        if dotenvy::var("SMTP_TLS")?.parse::<bool>().unwrap_or(true) {
+            Tls::Wrapper(TlsParameters::new(host.clone())?)
+        } else {
+            Tls::None
+        };
```

Comment:
Would you be open to extending the range of values accepted by `SMTP_TLS` to include both optional and required STARTTLS? This would provide full coverage of [`lettre`'s `Tls` enum](https://docs.rs/lettre/latest/lettre/transport/smtp/client/enum.Tls.html) and improve compatibility with services like Mailtrap, which rely on STARTTLS for encryption in transport.

I'd like to suggest the following values for `SMTP_TLS`, then: `none`, `opportunistic_starttls`, `required_starttls`, and `tls`.